README.md

```diff
@@ -58,6 +58,15 @@ Each Markdown file is associated with a Confluence wiki page with a Markdown com
 
 The above tells the tool to synchronize the Markdown file with the given Confluence page ID. This implies that the Confluence wiki page must exist such that it has an ID. The comment can be placed anywhere in the source file.
 
+### Provide generated-by prompt text in Markdown file
+
+Provide generated-by prompt text in Markdown file via the following optional tag:
+
+```markdown
+<!-- generated-by: Do not edit! Origin can be found <a href="https://git.example.com/project">here</a>. -->
```

Comment:
I am afraid I don't like this approach. At Instructure (the company I work for), we typically have a single generic text for all documents, e.g. all documents exported from the same Git repository have the same label that points to the Git repository root. I believe this generalizes to other companies and use cases, you don't typically want to set this on a per-document basis. As such, I believe a separate configuration file would be more useful; you configure the text once, and apply the text to a bunch of Markdown documents.

Reply:
Just to get it right: you think of a global configuration file (e.g. `~/.config/md2conf/config.yaml`) containing something in the sense:
```
---
generated-by: "This is my generic text for all documents."
```
I explicitly would like to define it per-document as I have several Markdown files laying around in different repositories where I would like to have a pointer (link) to these repositories in my generated Confluence pages.

Moreover, I don't think this is an either-or question, right? Why not providing all possibilities? One could define it per-document, or in a configuration file or via parameter. Precedence could be: parameter > per-document > global configuration file. Remember, this `generated-by` tag is optional.

Reply:
Yeah, that's a fair point. Let's keep the per-document tag too.

Reply:
Still my question: do you plan to have this kind of configuration file for md2conf (this should happen in a separate PR anyway)?

Reply:
Alright. Then feel free to merge this PR.